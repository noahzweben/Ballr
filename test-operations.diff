1c1,8
<   ./test-operations failed on ./test-operations
---
> 3
> -1
> 2
> 50
> 1
> -5
> SDL could not initialize! SDL Error: No available video device
> Window initialization failed.
