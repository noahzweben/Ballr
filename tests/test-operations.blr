gameboard g1{
	clr = (251,142,74);
	size = (680,420);
	init -> {

		print(1 + 2);
 		print(1 - 2);
  		print(1 * 2);
  		print(100 / 2);
  		print(99);

  	}
