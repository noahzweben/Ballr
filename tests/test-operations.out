3
-1
2
50
1
SDL could not initialize! SDL Error: No available video device
Window initialization failed.
